qlib/contrib/model/pytorch_utils.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import numpy as np
+import torch.nn as nn
+
+
+def count_parameters(model_or_parameters, unit="mb"):
```

Comment:
From my perspective, it is not necessary to make this feature as a single function, which actually can be implemented in one line. Here are some [resources](https://pytorch.org/docs/stable/generated/torch.numel.html) and references ([link1](https://discuss.pytorch.org/t/how-do-i-check-the-number-of-parameters-of-a-model/4325/25), [link2](https://stackoverflow.com/questions/49201236/check-the-total-number-of-parameters-in-a-pytorch-model)) may be useful.

If you really want to make it a decent feature, I suggest you can print the number of parameters in a nicer way (for example, similar to `Keras`'s stdout). But this should be carefully design, otherwise it will make the output very messy.